Vic2ToHoI4Tests/Vic2WorldTests/Ai/AIStrategy.cpp

```diff
@@ -0,0 +1,28 @@
+#include "../Vic2ToHoI4/Source/V2World/Ai/AIStrategy.h"
```

Comment:
It looks like you've just copied the file from Vic2ToHoI4. While you do need to include the cpp in the test project (add existing item under Vic2ToHoI4 files/Vic2), this should be a file called AIStrategy.cpp and structured like Vic2WorldTests/Provinces/ProvinceTests.cpp

Reply:
I'm not sure what I'm actually testing?